src/main/scala/dpla/ingestion3/enrichments/ParseDateEnrichment.scala

```diff
@@ -5,12 +5,15 @@ import java.text.SimpleDateFormat
 import scala.annotation.tailrec
 import scala.util.{Failure, Success, Try}
 import ParseDateEnrichment._
+import dpla.ingestion3.model.EdmTimeSpan
 
 class ParseDateEnrichment {
 
   //TODO ranges
-  def parse(dateString: String, allowInterval: Boolean = false): Option[String] = {
+  def parse(edtfOriginalDate: EdmTimeSpan, allowInterval: Boolean = false): EdmTimeSpan = {
+    val dateString = edtfOriginalDate.originalSourceDate.get
```

Comment:
Is `originalSourceDate` an Option?  If so, does this need to be a `getOrElse` or something in case the value is None?